docs/site/src/pages/style/color/color.md

```diff
@@ -0,0 +1,15 @@
+# Colors
```

Comment:
I would that that the closer we are from the specification, the simpler and the better.
`Color` sounds better https://material.io/guidelines/style/color.html